source/base/function_lib.cc

```diff
@@ -2947,6 +2947,34 @@ namespace Functions
            sizeof(coefficients);
   }
 
+  template <int dim>
+  RayleighKotheVortex<dim>::RayleighKotheVortex(const double T)
+    : Function<dim>(dim)
+    , T(T)
+  {
+    AssertThrow(dim > 1, ExcNotImplemented());
+  }
+
+
+  template <int dim>
+  void
+  RayleighKotheVortex<dim>::vector_value(const Point<dim> &point,
+                                         Vector<double> &  values) const
+  {
+    const double t = this->get_time();
+
+    const double px = numbers::PI * point(0);
+    const double py = numbers::PI * point(1);
+    const double pt = numbers::PI / T * t;
```

Comment:
Not sure if that's the best variable name. pt for me mentally translates to point... but maybe that's just me :)

Reply:
I am fine with the names.

Reply:
Maybe
```suggestion
    const double pi_t = numbers::PI / T * t;
```
and similarly above?

Reply:
@masterleinad I went with your name suggestion. Thanks :+1: